packages/generator/src/generation/context.ts

```diff
@@ -0,0 +1,9 @@
+import { TableType, UiFramework } from '../definition/context-types'
+import { Entity } from "./entity"
+
+export default interface GenerationContext {
+    useFormatter: boolean
+    tableType: TableType
```

Comment:
I am thinking about documenting reasonable deafaults. We might want specify Options data type and an instance with default values.